### ISSUE TITLE
Questions with respect to the code implementation

### ISSUE THREAD

**Issue by d12306** [xujinfan/Reinforced-Continual-Learning#3]
Hi, @xujinfan , thank you for your implemenation first. After checking the code, I find there are some questions in the implementation of the policy generator. Such as in the line 47 in policy_gradient.py, 
`self.loss =  -tf.log(picked_action_prob)*self.target`
the loss function of the RNN is the multiplication of the advantage and the probabilities of all the selected actions. 
```
 baseline_value = self.value_estimator.predict(self.state, self.sess)
 advantage = reward - baseline_value
```
Intuitively, the advantage is going to decrease in order to make the loss funciton decrease, but we eventually want the reward to become much larger. Am I misunderstanding your idea in the paper, could you please explain this more detailedly?

Thanks,

**Comment by xujinfan** [comment]
Hi，Xuefeng
The baseline_value is utilized for training the policy network more stably. You can search about A2C, a policy gradient algorithom, to learn more about it.

**Comment by d12306** [comment]
Thank you so much!

**Comment by d12306** [comment]
@xujinfan , Hi, sorry for bothering, but I feel strongly that we can change the input of the value predictor to the action-value or at least state-action value to approximate the Q value because the state here does not make any sense.
